test/runtests.jl

```diff
@@ -299,7 +309,8 @@ macro test_enumerate(input)
 end
 
 @test_enumerate [:a, :b, :c]
-@test_enumerate []
+@test_enumerate Union{}[]
+@test_enumerate Any[]
```

Comment:
Yep. Just being more explicit as these tests are no longer just meant to test "empty arrays".